Apps/Database/src/Delegates/EntityEmailDelegate.cs

```diff
@@ -22,18 +22,17 @@ namespace HealthGateway.Database.Delegates
     using HealthGateway.Database.Constants;
     using HealthGateway.Database.Context;
     using HealthGateway.Database.Models;
-    using Microsoft.EntityFrameworkCore;
 
     /// <inheritdoc />
-    public class EmailDelegate : IEmailDelegate
+    public class EntityEmailDelegate : IEmailDelegate
```

Comment:
The concrete implementations of our delegates should be "DB" or something rather than Entity.

Reply:
updated